indico/modules/designer/blueprint.py

```diff
@@ -35,6 +36,10 @@ def view_func(**kwargs):
 _bp.add_url_rule('/category/<int:category_id>/manage/designer/<int:template_id>/toggle-default-badge',
                  'toggle_category_default_badge', RHToggleBadgeDefaultOnCategory, methods=('POST',))
 
+_bp.add_url_rule('/event/<int:event_id>/manage/designer/<int:template_id>/link/<int:reg_form_id>', 'link_regform',
+                 RHLinkDesignerTemplate, defaults={'object_type': 'event'}, methods=('PATCH',))
+_bp.add_url_rule('/event/<int:event_id>/manage/designer/<int:template_id>/unlink', 'unlink_regform',
+                 RHUnLinkDesignerTemplate, defaults={'object_type': 'event'}, methods=('PATCH',))
```

Comment:
```suggestion
_bp.add_url_rule('/event/<int:event_id>/manage/designer/<int:template_id>/link/<int:reg_form_id>', 'link_regform',
                 RHLinkDesignerTemplate, defaults={'object_type': 'event'}, methods=('POST',))
_bp.add_url_rule('/event/<int:event_id>/manage/designer/<int:template_id>/unlink', 'unlink_regform',
                 RHUnLinkDesignerTemplate, defaults={'object_type': 'event'}, methods=('POST',))
```

"Non-RESTful" actions usually use POST. It's strange to use PATCH without a payload indicating what to update